docs/guide/performance.md

```diff
@@ -57,10 +57,11 @@ Barrel files are files that re-export the APIs of other files in the same direct
 // src/utils/index.js
 export * from './color.js'
 export * from './dom.js'
+export * from './slash.js'
 export * from './string.js'
```

Comment:
Good catch, we even have ``import { slash } from './utils/slash.js'`` example at the bottom.

I think we can remove this line as well then.
```suggestion
```